db/migrate/20220912100523_create_halls.rb

```diff
@@ -0,0 +1,10 @@
+class CreateHalls < ActiveRecord::Migration[7.0]
+  def change
+    create_table :halls do |t|
+      t.integer :capacity
```

Comment:
I think halls should always have capacity etc, so I suggest to add `null: false`